integrations/spark/spark-3.1/openhouse-spark-runtime/src/main/scala/com/linkedin/openhouse/spark/sql/catalyst/constants/Principal.scala

```diff
@@ -0,0 +1,18 @@
+package com.linkedin.openhouse.spark.sql.catalyst.constants
+
+/**
+ * This object is used to represent keyword global user group "PUBLIC" which maps to the acl policy representation "*"
+ */
+object Principal {
+  private val GLOBAL_USER_GROUP = "PUBLIC"
+  private val GLOBAL_USER_GROUP_ACL = "*"
+  def apply(principal: String): String = principal match {
+    case GLOBAL_USER_GROUP => GLOBAL_USER_GROUP_ACL
```

Comment:
What happens if we enter `public` instead `PUBLIC`?

Reply:
Oh this wouldn't be caught then, good catch. I will add handling for that.

Reply:
nit: GLOBAL_USER_GROUP_ACL could be renamed to GLOBAL_USER_GROUP_EXPR